src/gui/transferlistmodel.cpp

```diff
@@ -149,22 +149,22 @@ QVariant TransferListModel::headerData(int section, Qt::Orientation orientation,
             case TR_RATIO:
             case TR_PRIORITY:
             case TR_LAST_ACTIVITY:
-                return QVariant(Qt::AlignRight | Qt::AlignVCenter);
+                return {Qt::AlignRight | Qt::AlignVCenter};
```

Comment:
@thalieht 
This line introduced a warning: 
https://ci.appveyor.com/project/qbittorrent/qbittorrent/builds/22423194#L541

Could you revert it?

Reply:
Sure.

Reply:
Reverted in #10314